tests/cli/set/test_families.py

```diff
@@ -9,14 +9,14 @@
 SUCCESS = 0
 
 
-@pytest.mark.parametrize("identifier_key", ["ticket_number", "order"])
+@pytest.mark.parametrize("identifier_key", ["original_ticket", "order"])
 def test_set_families_by_sample_identifiers(
     cli_runner, base_context: CGConfig, identifier_key, helpers, caplog
 ):
     # GIVEN a database with a case with a sample
     base_store: Store = base_context.status_db
     sample_obj: models.Sample = helpers.add_sample(base_store)
-    sample_obj.ticket_number = 123456
+    sample_obj.original_ticket = "123456"
```

Comment:
I have seen this twice, "123456"! Make it a test_fixture - or reuse the ticket fixture

Reply:
Used the fixture instead